pytorch_lightning/metrics/classification/confusion_matrix.py

```diff
@@ -24,9 +24,11 @@
 
 class ConfusionMatrix(Metric):
     """
-    Computes the confusion matrix. Works with binary, multiclass, and multilabel data.
-    Accepts logits from a model output or integer class values in prediction.
-    Works with multi-dimensional preds and target.
+    Computes the `confusion matrix
+    <https://en.wikipedia.org/wiki/Confusion_matrix>`_.  Works with binary,
+    multiclass, and multilabel data.  Accepts logits from a model output or
+    integer class values in prediction.  Works with multi-dimensional preds and
+    target.
```

Comment:
An issue came up before about how the confusion matrix metric can't be directly logged like the other metrics since it outputs an array, should we add a reminder here as well?

Reply:
we could also add a plot function and the log this as an image?